### DISCUSSION TITLE
i have around seven eight folders in taskbar . how to sort / rearrange ?

### DISCUSSION THREAD

**Discussion by eternallearnings1** [valinet/ExplorerPatcher#3743] [Ideas]
i have around seven eight folders in taskbar . how to sort / rearrange ?

**Comment by pyrates999** [comment]
by drag and drop

**Comment by eternallearnings1** [comment]
[image: image.png]
Unable to drag and move. in the past i remember i could drag a folder and
move.
Please guide.

**Reply by pyrates999** [reply]
you can't attach images by replying via email.

**Comment by eternallearnings1** [comment]
How to describe the issue ?

**Reply by pyrates999** [reply]
Log into github and reply here.

**Comment by eternallearnings1** [comment]
Cannot move the folder panes as they seem locked.

**Reply by eternallearnings1** [reply]
![image](https://github.com/user-attachments/assets/0c8d85e5-b784-4b00-93b2-ae555cb60016)
i am unable to move the sections by dragging or dropping
Overall i created around 8 sections. i need to keep moving them around

**Reply by pyrates999** [reply]
This is what you've done:

https://www.lifewire.com/add-quick-launch-toolbar-in-windows-10-5115231

Except instead of creating one quick launch toolbar, you've created 8 of them.  Each quick launch toolbar has its own folder that it stores one or more folder shortcuts.

You can only move the folder shortcut within its own quick launch toolbar.

In order to move a folder shortcut to another quick launch toolbar, you must open the folder of each quick launch toolbar.  Then on the folder shortcut you want to move, right click on it, and select cut.  Then in the folder of the quick launch toolbar you want to move it to, right click and select paste.  Or use the drag and drop method.

You will not be able to do this directly from the windows 10 taskbar.  Even in windows 11.

If you do not see the change right away, you may have to restart explorer.exe using either task manager by killing explorer.exe and running it again, or opening up EP properties application and clicking on Restart File Explorer in the bottom left corner.

**Comment by eternallearnings1** [comment]
![image](https://github.com/user-attachments/assets/ff28c0c5-ab4f-4955-ad80-398c800914e8)
i convert into "Show title" format
then i am able to move this moving this from downwards. 
Some how works in awkward way 

Thanks
